openage/convert/hardcoded/texture.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+"""
+Constants for texture generation.
+"""
+
+MAX_TEXTURE_DIMENSION = 4096
+"""
+The maximum texture dimension that can be expected to be supported by all GPUs.
+"""
+
+MARGIN = 1
+"""
+A margin between subtextures in an atlas to avoid texture bleeding.
+"""
```

Comment:
the docstring should be above the variable it's for, and i'd simply use `#` comments here as they're not classes or functions.